lib/session.js

```diff
@@ -10,19 +10,17 @@ const createSession = async () => {
   const session = await db._client.startSession();
 
   const abortTransaction = async () => {
-    let error;
+    const errors = [];
     await session.abortTransaction();
     await session.endSession();
     await forEachSerialP(async (afterAborted) => {
       try {
-        await afterAborted(error);
+        await afterAborted();
       } catch (err) {
-        if (!err) {
-          error = err;
-        }
+        errors.push(err);
       }
     }, afterAborts);
-    if (error) { throw error; }
+    return errors;
```

Comment:
lets return `{ errors }` as a bit of future proofing (and the other function too)